### PR TITLE
BETA: Register berzcodetechnologies.is-a.dev

### PR DESCRIPTION
**Pull request by Sxurabh**
Added `berzcodetechnologies.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/berzcodetechnologies.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "BerzCode",
+        "email": "saurabhkirve@gmail.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=berzcodetechnologies.is-a.dev,e09feacd7133e558539a"
```
Comment (WilliamDavidHarrison) [inline domains/berzcodetechnologies.json:7]: We don't support Vercel.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/berzcodetechnologies.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "owner": {
+        "username": "BerzCode",
+        "email": "saurabhkirve@gmail.com"
+    },
+    "record": {
+        "A": "["154.41.233.131"]",
```
Comment (WilliamDavidHarrison) [inline domains/berzcodetechnologies.json:7]:
```suggestion
        "A": ["154.41.233.131"],
```

File: domains/saurabhkirve.json [hunk 3]
```diff
@@ -4,6 +4,6 @@
         "email": "saurabhkirve@gmail.com"
     },
     "record": {
-        "MX": ["mx1.hostinger.in"]
+        "A": "["154.41.233.131"]"
```
Comment (WilliamDavidHarrison) [inline domains/saurabhkirve.json:7]:
```suggestion
        "A": ["154.41.233.131"]
```

File: domains/berzcodetechnologies.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        "TXT": [
+            {
+                "name": "_github-pages-challenge-berzcode",
+                "value": "bafe74203bd3e05bb01b98f65606ab"
+            }
+        ]
```
Comment (WilliamDavidHarrison) [inline domains/berzcodetechnologies.json:13]: Your GitHub Pages verification TXT must be in a new file with the name GitHub provides you with a TXT as a string with the other value GitHub gives you.

Comment (WilliamDavidHarrison) [inline domains/berzcodetechnologies.json:13]: Your GitHub Pages verification TXT must be in a new file with the name GitHub provides you with a TXT as a string with the other value GitHub gives you.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — APPROVED** [review]

**Pull request merged**